src/layouts/Media/Media.tsx

```diff
@@ -325,62 +371,56 @@ export const Media = (): JSX.Element => {
             </VStack>
 
             {/* Action buttons segment */}
-            {(subDirCount !== 0 || filesCount !== 0) && (
+            {selectedMedia.length > 0 && (
               <>
                 <Spacer />
 
-                {selectedMedia.length > 0 && (
-                  <>
-                    <Box mt="auto">
-                      <Button
-                        variant="clear"
-                        color="base.content.strong"
-                        onClick={() => setSelectedMedia([])}
-                      >
-                        Deselect all
-                      </Button>
-                    </Box>
-                    <Box mt="auto">
-                      <Menu
-                        isStretch={false}
-                        onOpen={() => setIndividualMedia(null)}
+                <Box mt="auto">
```

Comment:
Indentation changes, mainly just adjusted the conditions in which the action buttons will appear.